src/components/ProductCard.jsx

```diff
@@ -32,7 +48,7 @@ const cartData = { _id:_id,
         </div>
         <div className="card-badge card-badge-position">{category}</div>
         <div className="favourite-icon favourite-icon-position">
-          <a className="btn btn-link-noBorder" href>
+          <a className="btn btn-link-noBorder" href="#" onClick={()=> addToWish(wishData)}>
```

Comment:
any reason for using a tag here

Reply:
It was not intentional I will make the changes right away